### ISSUE TITLE
TTS HTTP API response clarity

### ISSUE THREAD

**Issue by granolaonicecream** [rhasspy/rhasspy-server-hermes#17]
Minor issue I ran into while interfacing with the API.
In previous versions of Rhasspy, HTTP endpoint `text-to-speech` query parameter `play` would determine the response type:

- `true` -> returns string of the sentence played (equal to POST data)
- `false` -> returns wav data of what would have been played

The [logic now](https://github.com/rhasspy/rhasspy-server-hermes/blob/96593a50e6f2c780d5464fa7c93a0a31165ed306/rhasspyserver_hermes/__main__.py#L1585) is a bit stranger.  Regardless if anything is actually played, the logic is equivalent to:
```
if any(x == core.site_id for x in site_ids):
  audio is captured
  returns captured wav
else
  returns POST data sentence
```

Notably, wav data will always be returned if query param `siteId` is not provided.  This is a breaking change for clients like [rhasspy-client](https://github.com/rhasspy/rhasspy-client/blob/master/rhasspyclient/__init__.py#L204) that aren't expecting binary data.  Need to know if this is intentional in order to make a PR to either this repo or to rhasspy-client.

**Comment by synesthesiam** [comment]
Yeah, sorry about the confusing logic here. This is due to how satellites work in Rhasspy. The base station can send TTS messages out to its satellites, but it usually doesn't subscribe to the messages carrying the generated WAV audio.

So what I did was have the base station catch any audio *it* generates and return that. But if you only send TTS messages to satellites, you won't get any audio back, so I just returned the text.

Maybe I should just return empty WAV audio?
